Src/AutoFixture/RandomNumericSequenceGenerator.cs

```diff
@@ -53,7 +51,13 @@ public RandomNumericSequenceGenerator(params long[] limits)
 
             if (limits.Length < 2)
             {
-                throw new ArgumentException("The limit must be a sequence of two or more integers.");
+                throw new ArgumentException("Limits must be at least two ascending numbers.", "limits");
+            }
+
+            //Check that limits are ascending
+            if (limits.Zip(limits.Skip(1), (a, b) => a >= b).Any(b => b))

```

Comment:
This looks good! – Would it make sense to extract this into a function? Then, [we wouldn't need the comment](http://butunclebob.com/ArticleS.TimOttinger.ApologizeIncode), right? :smiley: